services/utils/conditional-headers/src/main/java/org/eclipse/ditto/services/utils/headers/conditional/ConditionalHeadersValidator.java

```diff
@@ -95,13 +107,36 @@ public void checkConditionalHeaders(final Command command,
 
         checkIfMatch(command, currentETagValue);
         checkIfNoneMatch(command, currentETagValue);
+
     }
 
     private boolean skipPreconditionHeaderCheck(final Command command, @Nullable final EntityTag
             currentETagValue) {
         return currentETagValue == null &&
                 (Command.Category.DELETE.equals(command.getCategory()) ||
-                        Command.Category.QUERY.equals(command.getCategory()));
+                        Command.Category.QUERY.equals(command.getCategory())) || skipDueToRelation(command);
+    }
+
+    /**
+     * Skip precondition check if the selected fields contain exempted fields (e.g. {@code _policy} for things
+     * because the revision of a thing does not change if its policy is updated).
+     */
+    private boolean skipDueToRelation(final Command command) {
+        return command.toJson().getValue(SELECTED_FIELDS)
+                .filter(JsonValue::isString)
+                .map(JsonValue::asString)
+                .map(str -> JsonFactory.newFieldSelector(str, JsonFactory.newParseOptionsBuilder()
+                        .withoutUrlDecoding()
+                        .build()))
+                .map(JsonFieldSelector::getPointers)
+                .map(this::containsExemptedField)
+                .orElse(false);
+    }
+
+    private boolean containsExemptedField(final Set<JsonPointer> selectedFields) {
+        final HashSet<JsonPointer> result = new HashSet<>(EXEMPTED_FIELDS);
```

Comment:
we don't use the concrete `HashSet` as type for variables, but `Set` instead. That way, the implementation can easily be modifed to e.g. another `Set` implementation.